tests/fuzz/runFuzzTest.sh

```diff
@@ -84,8 +84,11 @@ if [ "$CORPUS_UPLOAD_LOC" != "none" ]; then
         # The LD variables interferes with certificate validation when communicating with AWS S3.
         unset LD_PRELOAD
         unset LD_LIBRARY_PATH
-        printf "Copying corpus files from S3 bucket...\n"
-        aws s3 sync $CORPUS_UPLOAD_LOC/${TEST_NAME}/ "${TEMP_CORPUS_DIR}"
+        if aws s3 ls "${CORPUS_UPLOAD_LOC}/${TEST_NAME}/corpus.zip" > /dev/null 2>&1; then
```

Comment:
Can you add a comment explaining what this line is doing?

It looks like we call `ls` on an expected file and also suppress all terminal output? So `ls` returns a failure code if the file isn't found? But definitely a hairy piece of bash to parse 😓